ghdocs/CONTRIBUTING.md

```diff
@@ -29,14 +29,15 @@ This needs to only be done once for any Microsoft open source project.
 
 We track community issues via GitHub's [issue tracker](https://github.com/OfficeDev/Office-UI-Fabric/issues). To help us keep track of the different areas, please use the following labels:
 
-- `question` - Feel free to ask questions via the question tag.
-- `browser` - Issues that pertain to a specific browser. Screenshots and details are much appreciated!
-- `docs` - Issues pertaining to our documentation.
-- `feature` - Issues requesting a new feature to be added or extended from an existing feature.
-- `gulp` - Issues related to our gulp build process.
-- `js` - Issues that result from the example JavaScript referenced or included.
-- `meta` - issues with the overall project or the Github repository.
-- `components` - Issues specific to our CSS components.
-- `grid` - Issues pertaining to the responsive grid.
-- `type` - Issues pertaining to our typography and related classes.
-- `icons` - Issues pertaining to our iconography and icon related classes.
+ - `accessibility` - Issues or PRs that pertain to keyboard support, high contrast mode, screen reader support, etc.
+ - `bug` - Issues or PRs that address unexpected behavior or broken components.
+ - `build-system` - Issues and PRs related to the build process.
+ - `documentation` - Issues and PRs that pertain to the documentation included with the repository.
+ - `enhancement` - Issues or PRs that address potential improvements to components.
+ - `feature-request` - Issues or PRs that address ask for or add a major piece of functionality.
```

Comment:
Good catch, fixed.

From: Phil Worthington [mailto:notifications@github.com]
Sent: Monday, July 25, 2016 12:34 PM
To: OfficeDev/office-ui-fabric <office-ui-fabric@noreply.github.com>
Cc: Eric Thompson <boot@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [OfficeDev/office-ui-fabric] Updates CONTRIBUTING.md with proper labels (#741)


In ghdocs/CONTRIBUTING.md<https://github.com/OfficeDev/office-ui-fabric/pull/741#discussion_r72129155>:

> -- `browser` - Issues that pertain to a specific browser. Screenshots and details are much appreciated!

> -- `docs` - Issues pertaining to our documentation.

> -- `feature` - Issues requesting a new feature to be added or extended from an existing feature.

> -- `gulp` - Issues related to our gulp build process.

> -- `js` - Issues that result from the example JavaScript referenced or included.

> -- `meta` - issues with the overall project or the Github repository.

> -- `components` - Issues specific to our CSS components.

> -- `grid` - Issues pertaining to the responsive grid.

> -- `type` - Issues pertaining to our typography and related classes.

> -- `icons` - Issues pertaining to our iconography and icon related classes.

> + - `accessibility` - Issues or PRs that pertain to keyboard support, high contrast mode, screen reader support, etc.

> + - `bug` - Issues or PRs that address unexpected behavior or broken components.

> + - `build-system` - Issues and PRs related to the build process.

> + - `documentation` - Issues and PRs that pertain to the documentation included with the repository.

> + - `enhancement` - Issues or PRs that address potential improvements to components.

> + - `feature-request` - Issues or PRs that address ask for or add a major piece of functionality.

"...address, ask for, or add..." 👮

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/OfficeDev/office-ui-fabric/pull/741/files/d3501ec1fd2f64bc305c59a224042784781d2da4#r72129155>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ABWuNnSStXv0gxeqFwZVtGPTgufKPlYSks5qZQ-rgaJpZM4JTtgJ>.